.changeset/less-next-things.md

```diff
@@ -0,0 +1,5 @@
+---
+'@keystone-6/core': minor
```

Comment:
```suggestion
'@keystone-6/core': major
```

Although the flag is `experimental`, so I am marking the change as `major` as this will be releasing with other major changes, and it will easily be seen by anyone reviewing the CHANGELOG.